baseclasses/chemical_energy/potentiometry_gaschromatography.py

```diff
@@ -0,0 +1,478 @@
+                suggestions=[
+                    'PiperION',
+                    'SustainION',
+                    'Fumasep',
+                    'Nafion'
+                ])))
+
+    membrane_thickness = Quantity(
+        type=np.dtype(np.float64),
+        unit=('um'),
+        a_eln=dict(component='NumberEditQuantity', defaultDisplayUnit='um', props=dict(minValue=0)))
+
+    gasket_thickness = Quantity(
+        type=np.dtype(np.float64),
+        unit=('um'),
+        a_eln=dict(component='NumberEditQuantity', defaultDisplayUnit='um', props=dict(minValue=0)))
+
+    anolyte_type = Quantity(
+        type=str,
+        shape=[],
+        a_eln=dict(
+            component='EnumEditQuantity',
+            props=dict(
+                suggestions=[
+                    'KOH',
+                    'KHCO3'
+                ])))
+
+    anolyte_concentration = Quantity(
+        type=np.dtype(np.float64),
+        unit=('mol/L'),
+        a_eln=dict(component='NumberEditQuantity', defaultDisplayUnit='mol/L'))
+
+    anolyte_flow_rate = Quantity(
+        type=np.dtype(np.float64),
+        unit=('ml/minute'),
+        a_eln=dict(
+            component='NumberEditQuantity',
+            defaultDisplayUnit='ml/minute',
+            props=dict(minValue=0)
+        ))
+
+    anolyte_volume = Quantity(
+        type=np.dtype(np.float64),
+        unit=('ml'),
+        a_eln=dict(component='NumberEditQuantity', defaultDisplayUnit='ml'))
+
+    has_humidifier = Quantity(
+        type=bool,
+        default=True,
+        a_eln=dict(component='BoolEditQuantity')
+    )
+
+    humidifier_temperature = Quantity(
+        type=np.dtype(np.float64),
+        unit='°C',
+        a_eln=dict(component='NumberEditQuantity', defaultDisplayUnit='°C'))
+
+    water_trap_volume = Quantity(
+        type=np.dtype(np.float64),
+        unit=('ml'),
+        a_eln=dict(component='NumberEditQuantity', defaultDisplayUnit='ml'))
+
+    feed_gases = SubSection(
+        section_def=NECCFeedGas, repeats=True)
+
+    bleedline_flow_rate = Quantity(
+        type=np.dtype(np.float64),
+        unit=('ml/minute'),
+        a_eln=dict(
+            component='NumberEditQuantity',
+            defaultDisplayUnit='ml/minute',
+            props=dict(minValue=0)
+        ))
+
+    nitrogen_start_value = Quantity(
+        type=np.dtype(np.float64),
+        description='Specified in ppm',
+        a_eln=dict(component='NumberEditQuantity'))
+
+    # TODO Should an empty option also be specified? If excel is read and CP/CA is not set, CP is set automatically.
+    chronoanalysis_method = Quantity(
+        type=MEnum('Chronoamperometry (CA)', 'Chronopotentiometry (CP)'),
+        shape=[],
+        a_eln=dict(component='EnumEditQuantity'))
+
+    remarks = Quantity(
+        type=str,
+        a_eln=dict(
+            component='RichTextEditQuantity',
+            label="Remarks"
+        ))
+
+    def normalize(self, archive, logger):
+
+        # TODO experiment ID
```

Comment:
There is an experiment ID for the properties in excel. Do I convert this in the properties? Or do I use the lab_id of the superordinate measurement class for this?

Reply:
i wpuld but i in the `lab_id` field of the Measurement entry.

Reply:
Ok I tried that in https://github.com/nomad-hzb/nomad-baseclasses/pull/23/commits/d72304b22669616a8ea5786f63d9966c7662265d

Maybe check if it's ok to do it like this. I automatically generate it in the normalizer and I am not sure if it is ok to do it like that because I did not see it elsewhere in the code.

Reply:
yeah usually experiment dont have an extra id. only the cryptic entry_id but if they request it with an indivodual id this is fine